team_description_paper/RobotDescriptionDSPL.tex

```diff
@@ -9,7 +9,9 @@ \section{HSR's Software and External Devices}
 	\label{fig:hsr}
 \end{wrapfigure}
 
-We use a standard Toyota\texttrademark\hspace{0em} HSR robot. To differentiate our unit, we named it HERO. We wanted to link it's name to our AMIGO and SERGIO domestic service robots.
+We use a standard Toyota\texttrademark\hspace{0em} HSR robot. 
```

Comment:
Why all the stuff on the new lines? It is ignored by the compiler

Reply:
I know, but allows for easier outcommenting sentences that make stuff too long

Reply:
An better readability IMO